classes/task/emit_task.php

```diff
@@ -39,7 +39,7 @@ public function execute() {
         global $DB;
         $manager = get_log_manager();
         $store = new store($manager);
-        $extractedevents = $DB->get_records('logstore_xapi_log');
+        $extractedevents = $DB->get_records('logstore_xapi_log', null, '', '*', 0, $store->get_max_batch_size());
```

Comment:
Hey @AndyHubert, could you please extract these other arguments (`null`, `''`, `'*'`, and `0`) into variables so it's a little more obvious what they mean? For example the `'*'` might be defined as `$selectedfields = '*'` or something along those lines.